www/src/components/guidelines/system/text.js

```diff
@@ -1,7 +1,7 @@
 import styled from "@emotion/styled"
 import shouldForwardProp from "@styled-system/should-forward-prop"
```

Comment:
LGTM, but would be nice to get these `@styled-system` things out there, plus ideally `styled`. Would you be up to help with that, too?

Reply:
@fk yeah, I am working on it, I will raise the PR for all the components of styled-system used inside guidelines in one go or do you want me to do those changes in this same PR?

Reply:
🙏

Reply:
I could not get time to complete those changes so I thought let’s push these changes first and will do it in second go

Reply:
> do you want me to do those changes in this same PR?

However you want to do it;
lemme approve this one now!

Reply:
Thank you very much 🙏

Reply:
> > do you want me to do those changes in this same PR?
> 
> However you want to do it;
> lemme approve this one now!

Yes, I prefer doing it in second go